parser_connector.py

```diff
@@ -460,30 +460,33 @@ def handle_action(self, param):
     argparser.add_argument('input_test_json', help='Input Test JSON file')
     argparser.add_argument('-u', '--username', help='username', required=False)
     argparser.add_argument('-p', '--password', help='password', required=False)
+    argparser.add_argument('-v', '--verify', action='store_true', help='verify', required=False, default=False)
```

Comment:
reference: https://github.com/splunk-soar-connectors/microsoftteams/commit/c61cc62176d86230a52b289efc9992f28dcc7f68

This is great, as we're not modifying the verify behaviour!